src/coguard_cli/image_check/config_file_finders/config_file_finder_open_telemetry_collector.py

```diff
@@ -0,0 +1,180 @@
+"""
+This module contains the class to find OpenTelemetry Collector configurations
+inside a folder structure.
+"""
+
+import json
+import os
+import shutil
+import tempfile
+import logging
+from typing import Dict, List, Optional, Tuple
+from coguard_cli.image_check.config_file_finder_abc import ConfigFileFinder
+import coguard_cli.image_check.config_file_finders as cff_util
+from coguard_cli.print_colors import COLOR_CYAN, COLOR_TERMINATION
+
+class ConfigFileFinderOpenTelemetryCollector(ConfigFileFinder):
+    """
+    The class to find OpenTelemetry configuration files within a file system.
+    """
+
+    def _create_temp_location_and_mainfest_entry(
```

Comment:
Move this maybe to util?

Reply:
Impossible as there are some details which are unique to this case. Let's support more services before going this way.